testpilot/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -52,6 +52,11 @@ export default `
                       <td data-l10n-id="bugReports">Bug Reports</td>
                       <td><a data-hook="bug-report-url"></a></td>
                     </tr>
+
+                    <tr>
+                      <td data-l10n-id="discourse">Discourse</td>
```

Comment:
cc/ @johngruen Are you cool with this copy? Would you rather have something like "forum" or "discuss" here?